src/composables/loginApi.js

```diff
@@ -87,19 +110,21 @@ export const loginApi = () => {
                 router.push('/home')
             }
         } catch (error) {
-            console.log(error.message)
+            console.log(error)
+        } finally {
+            isLoading.value = false
         }
     }
 
     //for logout
     const logout = () => {
-        localStorage.setItem('user' , JSON.stringify(null))
-        localStorage.setItem('userToken' , JSON.stringify(null))
+        localStorage.setItem('user', JSON.stringify(null))
```

Comment:
at the time for logout remove items from local storage, do not set them null @PiyushInnotech